src/mon/Monitor.cc

```diff
@@ -3031,7 +3031,7 @@ void Monitor::get_cluster_status(stringstream &ss, Formatter *f)
 	ss << "    " << i.second.message << "\n";
 	ss << "      [";
 	unsigned j;
-	for (j=0; j < i.second.progress * 30; ++j) {
+	for (j = 0; j < (unsigned)(i.second.progress * 30.0); ++j) {
```

Comment:
nit, no need to use `30.0`. `i.second.progress * 30` will be a float anyway.

Reply:
Even if it's not needed, I actually like to have it there for clarity. This way we don't have to inspect `progress`'s type to understand, at a glance, that it will return a float.